packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -247,7 +247,7 @@ export default class InternalModel {
   get _recordData(): RecordData {
     if (this.__recordData === null) {
       let recordData = this.store._createRecordData(this.identifier);
-      this._recordData = recordData;
+      this.__recordData = recordData;
```

Comment:
This seems like...it was a bug??

Reply:
Causes an extra set but otherwise did work